commands/Tools/Step 3 - Modify schematisation/control_structures.py

```diff
@@ -1,37 +1,37 @@
 # -*- coding: utf-8 -*-
 # (c) Nelen & Schuurmans, see LICENSE.rst.
 
-from builtins import str
 from builtins import range
-import logging
-
+from builtins import str
```

Comment:
Are `builtins` imports necessary/pythonic?
Seems like they can all be removed. https://docs.python.org/3/library/builtins.html

Reply:
Done